pkg/apis/stork/v1alpha1/migration.go

```diff
@@ -25,6 +25,7 @@ type MigrationSpec struct {
 	PreExecRule                  string            `json:"preExecRule"`
 	PostExecRule                 string            `json:"postExecRule"`
 	IncludeOptionalResourceTypes []string          `json:"includeOptionalResourceTypes"`
+	SkipDeletedNamespaces        bool              `json:"skipDeletedNamespaces"`
```

Comment:
Does this default to False?
The other bools here are *bool

Reply:
Yes it will default to false